### PR TITLE
feat: Add text truncate with no ellipsis class - MEED-5901 -Meeds-io/MIPs#124

### PR DESCRIPTION
**Pull request by SaraBoutej**
Add new css class helper

### CODE REVIEW

**Review by boubaker — APPROVED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/core/helpers.less [hunk 1]
```diff
@@ -17,6 +17,12 @@
   Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+@font-face {
+  font-family: "ellipsis-font";
+  src: local('DejaVu Sans');
```
Comment (boubaker) [inline platform-ui-skin/src/main/webapp/skin/less/core/helpers.less]: Please rename font family and define the default fonts as defined in `@fontFamily` variable in order to preserve the font family of displayed text inside activities.

```suggestion
  font-family: "Hide Line Clamp Font";
  src:
    local('Helvetica')
    local('arial')
    local('sans-serif');
```

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/helpers.less]: fixed

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/helpers.less]: fixed

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/helpers.less]: fixed

**Review by boubaker — APPROVED** [review]

**Pull request merged**